packages/tables/resources/views/columns/toggle-column.blade.php

```diff
@@ -2,53 +2,34 @@
     $state = $getState();
 @endphp
 
+<div wire:key="{{ 'toggle-column-' . $recordKey . '-' . $getName() . '-' . json_encode($state) }}">
```

Comment:
Could you fix indentation, @archilex?

Reply:
Sure! Done. I also reformatted the`wire:key` to better follow Filament's existing style